### PR TITLE
[back] add uncertainties in the public dataset

### PR DESCRIPTION
**Pull request by aidanjungo**
#1397

### CODE REVIEW

**Comment by aidanjungo** [comment]
@lfaucon @amatissart  please check and improve what I wrote in `export_readme.txt`

File: backend/tournesol/lib/public_dataset.py [hunk 1]
```diff
@@ -302,6 +304,7 @@ def write_individual_criteria_scores_file(poll_name: str, write_target) -> None:
             "score": round(criteria_score.score, FLOAT_PRECISION),
             # The voting rights can be very small and reach numbers like 10e-3
             # or even 10e-4. Thus, we round them with more precision.
+            "uncertainty": round(criteria_score.uncertainty, FLOAT_PRECISION),
```
Comment (GresilleSiffle) [inline backend/tournesol/lib/public_dataset.py:307]: The comment above applies to the voting right, and should be displayed before the voting rights, not before uncertainty.

```suggestion
            "uncertainty": round(criteria_score.uncertainty, FLOAT_PRECISION),
            # The voting rights can be very small and reach numbers like 10e-3
            # or even 10e-4. Thus, we round them with more precision.
```

File: backend/tournesol/resources/export_readme.txt [hunk 2]
```diff
@@ -138,6 +138,13 @@ List of columns:
   are globally scaled such that they are meaningfully comparable and can be
   aggregated.
 
+- uncertainty:
+
+  This field is computed by Tournesol's algorithms. It should represent the 
+  uncertainty on the score value. Be careful, due to the score scaling which it
+  may seem very large. For more detail, check out the algorithms implementation 
+  or the paper which describe them https://arxiv.org/abs/2211.01179
```
Comment (lfaucon) [inline backend/tournesol/resources/export_readme.txt:146]:
```suggestion
  This field is computed by Tournesol's algorithms. It should represent the 
  uncertainty on the score value. For more detail, check out the algorithms
  implementation or the paper which describe them
  https://arxiv.org/abs/2211.01179
```

File: backend/tournesol/resources/export_readme.txt [hunk 3]
```diff
@@ -174,3 +181,10 @@ List of columns:
   individual scores. With a small number of contributors, this score would be
   reduced towards 0 such that it is not possible for a small number of users to
   give a very high collective score on a video.
+
+- uncertainty:
+
+  This field is computed by Tournesol's algorithms. It should represent the 
+  uncertainty on the score value. Be careful, due to the score scaling which it
+  may seem very large. For more detail, check out the algorithms implementation 
+  or the paper which describe them https://arxiv.org/abs/2211.01179
```
Comment (lfaucon) [inline backend/tournesol/resources/export_readme.txt:190]: Suggesting to remove the sentence that is not grammatically correct and does not help but confuse more the users. Let's do some analysis on this data and learn better how to document it.

```suggestion
  This field is computed by Tournesol's algorithms. It should represent the 
  uncertainty on the score value. For more detail, check out the algorithms
  implementation or the paper which describe them
  https://arxiv.org/abs/2211.01179
```

**Review by lfaucon — APPROVED** [review]

**Pull request merged**